installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc

```diff
@@ -57,29 +95,22 @@ include::modules/nw-operator-cr.adoc[leveloffset=+1]
-system so that it takes effect on first boot. In special cases, you can create a
-separate, limited Ignition config to pass to the live system. That Ignition config could do a certain set of tasks, such as reporting success to a provisioning system
-after completing installation. This special Ignition config is consumed by the installer and should not
-be used to include the standard `worker` and `master` Ignition configs.
+== Installing {op-system} and starting the {product-title} bootstrap process
+
+To install {product-title} on bare metal infrastructure that you provision, you must install {op-system-first} on the machines. When you install {op-system}, you must provide the Ignition config file that was generated by the {product-title} installation program for the type of machine you are installing. If you have configured suitable networking, DNS and load balancing infrastructure, the {product-title} bootstrap process will begin automatically after the {op-system} machines have rebooted.
+
+To install {op-system} on the machines, follow either the steps to use an ISO image or network PXE booting.
+
+[NOTE]
+====
+The compute node deployment steps included in this installation document are {op-system}-specific. If you choose instead to deploy {op-system-base}-based compute nodes, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks.
+====
+
+You can configure {op-system} during ISO and PXE installations by using the following methods:
+
+* Kernel arguments: You can use kernel arguments to provide installation specific information. For example, you can specify the locations of the {op-system} installation files that you uploaded to your HTTP server and the location of the Ignition config file for the type of node you are installing. For a PXE install, you can use the `APPEND` parameter to pass the arguments to the kernel of the live installer. For an ISO install, you can interrupt the live installation boot process to add the kernel arguments. In both cases, you can use special `coreos.inst.*` arguments to direct the installer, as well as standard boot arguments for turning standard kernel services on or off.
+
+* Ignition configs: {product-title} Ignition config files (`*.ign`) are specific to the type of node you are installing. You pass the location of a bootstrap, control plane, or compute node Ignition config file during the {op-system} installation so that it takes effect on first boot. In special cases, you can create a separate, limited Ignition config to pass to the live system. That Ignition config could do a certain set of tasks, such as reporting success to a provisioning system after completing installation. This special Ignition config is consumed by the installer and should not be used to include the standard `master` and `worker` Ignition configs.
```

Comment:
> This special Ignition config is consumed by the installer 

"installer" is ambiguous, I'd say something more like:

> ...consumed by `coreos-installer` to be applied on first boot of the installed system.  Do not provide the standard `master` and `worker` Ignition configs to the live ISO directly.

Reply:
@cgwalters thank you for your review and for this suggestion. I have now applied the suggestion to the four relevant assemblies in this PR.